docs/best-practices.md

```diff
@@ -66,6 +66,18 @@ Please see [Troubleshooting Complex URLs](https://asciidoctor.org/docs/user-manu
 This is an example of an URL containing problematic characters which needs special treatment:
 `https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)`
 
+**Strongly** in favour using following method which greatly improves readbility during authoring the document.
```

Comment:
I'd change this to "Strongly in favour of using the following method which greatly improves readability while authoring the document"

Reply:
With or without bold "Strongly" ?